client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.module.scss

```diff
@@ -13,5 +13,5 @@
 
 .add-button-item {
   padding-bottom: 20px;
-  padding-left: 20px;
+  padding-inline: var(--global-space--section-left);
```

Comment:
**Why?**
- To center the (full-width) button, instead of just push it to the left.[^1]

[^1]: I would also use only `margin` not `padding`, but both work here and I try to keep diff light.